src/library.js

```diff
@@ -344,6 +344,23 @@ const createPilingJs = (rootElement, initOptions = {}) => {
   const activePile = new PIXI.Container();
   const normalPiles = new PIXI.Container();
 
+  const addToActivePile = pileGfx => {
+    if (activePile.children.length) {
+      normalPiles.addChild(activePile.getChildAt(0));
+      activePile.removeChildren();
+    }
+    activePile.addChild(pileGfx);
+  };
+
+  const addToNormalPiles = pileGfx => {
```

Comment:
```suggestion
  const moveToNormalPileLayer = pileGfx => {
```

As before I think that's a bit clearer.

Also, is this function needed at all? In `moveToActivePileLayer()` the pileGfx is never removed from the normal pile layer. Hence, if we just clear `activePile` than shouldn't the `pileGfx` already be in `normalPiles`?

Reply:
Yeah, you're right 😂